azure-pipelines.yml

```diff
@@ -1,34 +1,40 @@
 # Azure DevOps pipeline to build, check source codes, run tests, and deploy.
 #
-# The following variables needs to be defined before running the deployment jobs 
-# defined in this pipeline:
+# To enable the check of source code with Danger JS you need to configure a valid
+# GitHub token by setting the following variable: 
```

Comment:
do we know the the required permission for this token ? (repo:*) ?

Reply:
Based to the documentation for public repository (as all IO repos) it is only necessary to give the account token the smallest scope possible (this means just _public_repo_). 

In the current Azure DevOps pipeline, I configured the variable _DANGER_GITHUB_API_TOKEN_ with the token that was provided to me by @cloudify (I do not know the associated permissions).